sys/net/gnrc/link_layer/gomach/include/gomach_internal.h

```diff
@@ -0,0 +1,928 @@
+/*
+ * Copyright (C) 2016 INRIA
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_gnrc_gomach
+ * @{
+ *
+ * @file
+ * @brief       GoMacH's internal functions.
+ * @internal
+ * @author      Shuguo Zhuo  <shuguo.zhuo@inria.fr>
+ */
+
+#ifndef GOMACH_INTERNAL_H
+#define GOMACH_INTERNAL_H
+
+#include <stdint.h>
+
+#include "periph/rtt.h"
+#include "net/gnrc/netif.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief Flag to track if the transmission has finished.
+ */
+#define GNRC_GOMACH_INFO_TX_FINISHED         (0x0008U)
+
+/**
+ * @brief Flag to track if a packet has been successfully received.
+ */
+#define GNRC_GOMACH_INFO_PKT_RECEIVED        (0x0010U)
+
+/**
+ * @brief Flag to track if need to update GoMacH.
+ */
+#define GNRC_GOMACH_INTERNAL_INFO_ND_UPDATE        (0x0001U)
```

Comment:
Here, above and below: please unify the the spacing between the macro and the value (i.e. so that the values are all in the same column)